js/packages/components/settings/DevTools.tsx

```diff
@@ -344,6 +347,40 @@ const BodyDevTools: React.FC<{}> = () => {
 					})
 				}}
 			/>
+			<ButtonSetting
+				name={t('settings.devtools.tor-required-button')}
+				icon='tor'
+				iconPack='custom'
+				iconSize={30}
+				iconColor={color.dark.grey}
+				varToggle={ctx.persistentOptions?.tor.flag === 'required'}
+				actionToggle={async () => {
+					await ctx.setPersistentOption({
+						type: PersistentOptionsKeys.Tor,
+						payload: {
+							flag: ctx.persistentOptions?.tor.flag === 'required' ? 'disabled' : 'required',
+						},
+					})
+				}}
+				toggled
+			/>
+			<ButtonSetting
+				name={t('settings.devtools.tor-optional-button')}
+				icon='tor'
+				iconPack='custom'
+				iconSize={30}
+				iconColor={color.dark.grey}
+				varToggle={ctx.persistentOptions?.tor.flag === 'optional'}
+				actionToggle={async () => {
+					await ctx.setPersistentOption({
+						type: PersistentOptionsKeys.Tor,
+						payload: {
+							flag: ctx.persistentOptions?.tor.flag === 'optional' ? 'disabled' : 'optional',
+						},
+					})
+				}}
+				toggled
+			/>
```

Comment:
Couldn't be a dropdown menu instead ?

Reply:
We decided that for the beginning to have 2 toggles for the beginning 👍